.github/labeler.yml

```
@@ -6,3 +6,6 @@
 "doc-required":
   - "doc/**/*"
   - "**/*.rst"
+
+"no-changelog"
```

```suggestion
"no-changelog":
```
No idea about the syntax, but I would assume it's the same as in the previous entry. ;)

---

> No idea about the syntax, but I would assume it's the same as in the other entries. ;)

Indeed, fixed now